WordPress/Docs/Arrays/ArrayDeclarationSpacingStandard.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Array Declaration Spacing"
+    >
+    <standard>
+    <![CDATA[
+    When an array uses associative keys, each pair of key and value must start on a new line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: There is only one pair of key and value per line.">
+        <![CDATA[
+$args = array(
```

Comment:
Aside from the issue the sniff is about, please check that the code in each code sample follows the WP Coding Standards. I believe this code example is missing a trailing comma after the last array item, and the alignment of the first double arrow is incorrect.